Tasks/AzureContainerAppsV0/task.json

```diff
@@ -0,0 +1,170 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Microsoft/azure-pipelines-task-lib/master/tasks.schema.json",
+    "id": "35f6376b-4761-4488-a921-f882b2c994bb",
+    "name": "AzureContainerApps",
+    "friendlyName": "Azure Container Apps Deploy",
+    "description": "An Azure DevOps Task to build and deploy Azure Container Apps.",
+    "author": "Microsoft Corporation",
+    "helpMarkDown": "[Learn more about this task](http://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/AzureContainerAppsV0/README.md)",
+    "releaseNotes": "Released new AzureContainerApps task for building and deploying Azure Container Apps.",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "version": {
+        "Major": 0,
+        "Minor": 1,
```

Comment:
Minor should be the number of sprint when this was introduced. Currently 213 https://whatsprintis.it/